prime-router/docs/getting-started/README.md

```diff
@@ -2,37 +2,172 @@
+The `cleanslate.sh` script does the base work needed to start developing for ReportStream. It only needs to be run once. This script runs on Apple processors, but it skips a few steps. We will need to do these missing steps by hand.
 
-### Running the static site
+1. [Clone the prime-reportstream repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository) to your workstation using git.
+
+1. Initialize your environment and run an initial build by running the following command using a Linux shell.
+   Note you can run `cleanslate.sh` script to reset your environment as well (run `./cleanslate.sh --help` for more
+   information). The `cleanslate.sh` script not only cleans, but also performs a first build and setup of all artifacts
+   needed for the build
+
+```bash
+cd ./prime-router
+./cleanslate.sh
+```
+
+<details>
+  <summary>Docker Desktop</summary>
+* If you are using Docker Desktop, verify that it is running prior to building or running ReportStream locally.
+</details>
+
+<details>
+  <summary>Gradle/Apple Silicon</summary>
+
+#### Run cleanslate.sh
+
+```bash
+# build the project
+./cleanslate.sh --verbose
+# ...
+
+# Check that a Postgres instance is running
+docker ps
+# CONTAINER ID   IMAGE         COMMAND                  CREATED          STATUS          PORTS                    NAMES
+# 2962fb214203   postgres:11   "docker-entrypoint.s…"   57 minutes ago   Up 57 minutes   0.0.0.0:5432->5432/tcp   prime-router_postgresql_1
+```
+
+#### Run support services
+
+ReportStream depends on set of services to be up before running the main Azure service. The `cleanslate.sh` script
+starts a Postgres database but skips starting a few more that are otherwise started by default when `cleanslate.sh`
+start is run on a non-Apple processor:
+
+- Azurite - a simulator of Azure storage
+- Vault - a secret store
+- SFTP - an SFTP server
+- soap-webservice - SOAP web service emulator
+
+```bash
+docker-compose -f docker-compose.build.yml up --detach
+```
+
+Additionally, to ensure that Vault is running and the credentials are stored correctly, run the following (which is
+normally covered by `cleanslate.sh` on non-Apple processors):
+
+```bash
+docker-compose up --detach vault 1>/dev/null 2>/dev/null
+```
+
+You can take down these services by running `./gradlew composeDown` or `docker-compose down` command.
+For now, leave these services running and open up a new terminal session.
+</details>
+
+### Seed the Postgres db and vault
+
+To run tests, the Postgres DB and the credential vault need to be seeded with values.
+We will need to have ReportStream running for these steps to work (see previous steps).
+Again, we will use a Gradle task to do these steps.
+
+```bash
+./gradlew primeCLI --args "create-credential --type=UserPass --persist=DEFAULT-SFTP --user foo --pass pass"
+./gradlew reloadTables
+./gradlew reloadSettings
+```
+
+## Running ReportStream backend
```

Comment:
Probably worth having a conversation around it, but I'm wondering if it might make sense to recommend to everyone to use `./gradlew quickRun` as the default for starting ReportStream.  I think some confusion in the past has resulted in much slower cycles from either using `./gradlew run` and waiting for all the tests or the container taking a long time to build

Reply:
Going to update it to recommend `quickRun`. 

While looking into this I think that the part I hid under Docker Desktop "If you are using Docker Desktop, verify that it is running prior to building or running ReportStream locally." The way I have my project setup I have to have Docker Desktop open and running for docker-compose to work. Am I alone in that or should I move that instruction to the main workflow.